gocourse02/go.mod

```
@@ -0,0 +1,5 @@
+module prjctr-golang_beginning
```

Дякую, що додав go.mod/go.sum.

Ось так більш ідіоматично буде:
```suggestion
module github.com/sarff/prjctr-golang_beginning/gocourse02
```

Далі на курсі дізнаємось, чому.